core/store/migrate/migrations/0232_log_poller_word_topic_indexes.sql

```diff
@@ -0,0 +1,62 @@
+-- +goose Up
+
+drop index if exists evm.evm_logs_idx_data_word_one;
+drop index if exists evm.evm_logs_idx_data_word_two;
+drop index if exists evm.evm_logs_idx_data_word_three;
+drop index if exists evm.evm_logs_idx_data_word_four;
+drop index if exists evm.evm_logs_idx_topic_two;
+drop index if exists evm.evm_logs_idx_topic_three;
+drop index if exists evm.evm_logs_idx_topic_four;
+
+create index evm_logs_idx_data_word_one
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 1, 32));
+
+create index evm_logs_idx_data_word_two
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 33, 32));
+
+create index evm_logs_idx_data_word_three
+    on evm.logs (evm_chain_id, address, event_sig, "substring"(data, 65, 32));
```

Comment:
Missed this before, but you need to either append `evm.` to all of these `create`'s here, to match the `drop`'s in the `Goose down` stage, or if you like we could remove the redundancy and replace `evm_` here with `evm.` and `evm.evm_` there with `evm.`.  (I guess then it wouldn't match other index names in the same schema though, I'm fine with it either way.)